svthalia/concrexit
# Create event URL slugs

Closes #2308 

### Summary
Create event URL slugs 

### How to test
Steps to test the changes you made:
1. Run the migrations
2. Observe the generated slugs
3. Create a new event
4. Observe the slug field and see the cool pre-population
5. Submit the form
6. Go to the website/events/slug
7. Also check if the backup (website/events/pk) still works
8. Everything works
